eZ/Publish/Core/FieldType/TextLine/Type.php

```diff
@@ -31,7 +31,7 @@ class Type extends FieldType
             ),
             'maxStringLength' => array(
                 'type' => 'int',
-                'default' => false
+                'default' => null
```

Comment:
Don't think the change will break anything. However, `validateValidatorConfiguration()` may report new errors if one is passing `false`.